Core/StitchCoreAdminClient/Sources/StitchCoreAdminClient/AuthProviders/ProviderConfig.swift

```diff
@@ -17,6 +17,13 @@ private enum UserpassCodingKeys: String, CodingKey {
     case resetPasswordURL = "resetPasswordUrl"
     case confirmEmailSubject
     case resetPasswordSubject
+    case runResetFunction
+    case resetFunctionId
+    case resetFunctionName
```

Comment:
is this PR also supposed to have reset function stuff in it? seems like this should be in a different ticket. If this is a driveby, shouldn't it also contain confirmation function stuff as well?